package-lock.json

```diff
@@ -5,6 +5,79 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "excalibur",
```

Comment:
I think this file will need to be reverted too, makes reference to your local Meet/Excalibur

Reply:
might need some guidance on this, i've never touched a package-lock.json

it's usually managed by node, right?

Reply:
i've deleted my package-lock and redid a `npm install` and it keeps coming up with local references in my package-lock...
@eonarheim